Documentation/bpf.rst

```diff
@@ -4619,22 +4624,22 @@ There are a couple of walk-through posts by David S. Miller to the xdp-newbies
 mailing list (http://vger.kernel.org/vger-lists.html#xdp-newbies), which explain
 various parts of XDP and BPF:
 
-4. May 2017,
+1. May 2017,
```

Comment:
Please keep the reverse order. If we ever change them we want to do it consistently on the whole page (i.e. other sections too), and this is beyond the scope of this PR.

Looking at the commit history, it looks like you reverted this change, but somehow reintroduced it (inadvertently?) in the last commit.

Reply:
Thank you will do so.

Reply:
Done.